content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -236,10 +232,17 @@ sudo rmdir /mnt/data
 
 You can now close the shell to your Node.
 
+<!-- discussion -->
 
+## Mounting the same persistentVolume in two places
 
+{{< codenew file="pods/storage/pv-duplicate.yaml" >}}
 
-<!-- discussion -->
+Example :
+We can perform 2 volume mounts on your nginx container:
```

Comment:
This format is strange. Please consider revise.

Reply:
is this the correct format?

Example : 
```
# You can perform 2 volume mounts on your nginx container : 
1.  /usr/share/nginx/html for the static website
2.  /etc/nginx/nginx.conf for the default config
```